HTML and CSS files are added to the mobile-portfolio branch. I am looking forward to get feedback on my process.

rozhi-98 opened the pull request:
**portfolio-mobile-first**

- Added header and headline sections of my portfolio like the design on figma.
- Added a folder which contains all the images that I will be using in the project.
- Updated my REDME file in a more descriptive and professional way.

rozhi-98 committed:
Add html file

rozhi-98 committed:
Add css file

rozhi-98 committed:
Add html code for header and headline sections

rozhi-98 committed:
Add css styles for header and headline sections

rozhi-98 committed:
Add images folder

andgarzonmal reviewed (changes requested):
Hi @rozhi-98 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, without a professional README file 

- [x] You have some Linters errors 

![image](https://user-images.githubusercontent.com/95243996/191953183-92cd0283-8c0a-447e-8e28-abbd5b5db9e8.png)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

rozhi-98 committed:
Remove stylelint errors from style.css file

rozhi-98 committed:
Update README file and include more information in it in a professional way

rozhi-98 commented:
I have fixed the issue with stylelint so all linters checks pass successfully now. I have also updated my README file with nmore information in a professional way.

Gopxfs commented:
# STATUS: INVALID REQUEST

Hi @rozhi-98 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that doesn't contain a description.
Please check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand more about the basics pull request rules at Microverse 😀.
My tip for you: try to give a short description about what you made on this project, maybe a summary. What features did you add? What is new on your project? Good luck!

Your Code Review Request will be marked invalid in your Dashboard, so please submit a new one once ready 🙏.
Invalid requests don't count as a review, so don't worry about that 😄!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,53 @@
+  <body>
+    <section class="container">
+      <header>
+        <img src="images/Light.png" alt="status bar" width="100%" />
+        <div class="toolbar">
+          <h1 id="logo">ROZH</h1>
+          <img id="menu-image" src="images/Icon - Menu.png" alt="Menu icon" />
+        </div>
```
- [x] Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. The image you added is unnecessary because this is what the user's phone will display and should not be displayed by the website, so you should remove that.
- [x] The logo should be wrapped in an `<a>` tag for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. Kindly change that.
- [x] The hamburger button should be wrapped in a `<nav>` tag for semantics reasons. In the future, you will implement functionality for this button to open a mobile menu but you can leave it as an image by now. I suggest you just wrap the hamburger button around the `<nav>` tag.

rozhi-98 replied:
Thanks so much for the feedback. I implemented all the suggestions.

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,53 @@
+
+      <section class="headline">
+        <article class="self-info">
+          <h2>
+            I'm Rozh <br />
+            Glad to see <br />
+            you! <br />
+          </h2>
```
- [x] This is the main header of the portfolio and should be wrapped in an `<h1>` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info about the importance of the `h1` tag. Kindly change that.

rozhi-98 replied:
Thanks so much for the feedback. I changed it to h1.

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,53 @@
+          </p>
+          <h3>LET'S CONNECT</h3>
+        </article>
+        <div class="social-media">
+          <img src="images/Vector.png" alt="Twitter Logo" />
+          <img src="images/Linkedin icon.png" alt="Linkedin Logo" />
+          <img src="images/Vector (1).png" alt="mobile" />
+          <img src="images/Vector (2).png" alt="Github Logo" />
+          <img src="images/Vector (3).png" alt="Hand Logo" />
+        </div>
```
- [x] This is a list of social media links, so the list would be better wrapped with a `ul` tag with each item wrapped by the `li` tag. Since it is a list of links, each link should be wrapped with the `a` tag. Please make these changes to make your HTML file semantically correct and don't forget to add the `href` attribute for each link (if you don't have a social media of any of these platforms you can reference this page by passing the `href="#"`  to link to the top of the current page).

rozhi-98 replied:
Thanks so much for the feedback. I implemented the suggestion but I had to remove the div that was surrounding the my social media icons and had a class of "social-media". Instead, I gave that class name to the ul where all my li and a tags are. is that ok? @leonardodiasb

bobb-Rob replied:
@rozhi-98, That is okay

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,53 @@
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;800&display=swap"
+      rel="stylesheet"
+    />
```
- [x] Great use of the 'Poppins' font. I see that you used different font-weights in your project that you did not import. I suggest you import all the font-weights used so they can be used and your portfolio will look more similar to the template you are following. (`<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">`)

rozhi-98 replied:
Thanks so much for the feedback. I changed the weights to meet the requirements.

leonardodiasb reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @rozhi-98, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 

 - Nice CSS styling. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rozhi-98 committed:
Update the html file to put all social media icons in a ul rather than a div and change the fnts weight

rozhi-98 committed:
Update styles for social-media class

rozhi-98 committed:
Fix stylelint error on having px on zero value

rozhi-98 committed:
Delete unwanted header image

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @rozhi-98 , 
You did well implementing all the suggestions by the previous reviewer. 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
